### DISCUSSION TITLE
MFXTextField transparent background strange behavior

### DISCUSSION THREAD

**Discussion by GrolarDan** [palexdev/MaterialFX#217] [Archived]
Hi, 

first i want to say I like your library very much.

I want to use dark background one my scene with MFXTextField which should have transparent background.
I'm able to set the style and it works.
```
#input {
    -fx-background-color: #00FF0022;
    -fx-text-fill: white;
}
```
But there is a strange behavior. When I'm writing text in the field the text background changes to color without the set opacity.
I tried to look into your code and read the wiki but without any luck how to solve it.
I tried same styling on the TextField and there it behaves as expected.

Here is github with my code. Tt is the `sec-04-login-dark` module.
https://github.com/GrolarDan/javafx-animation-material-design/tree/login-dark/sec-04-login-dark

I would really appreciate your advice.
Thanks

**Comment by palexdev** [comment]
@GrolarDan I've looked into the issue but I can't reproduce it.
Using your stylesheet everything seems to be working as expected. The only thing is the background shrinking/expanding when focused/unfocused but even that is normal and must be tweaked/fixed in css
